### PR TITLE
[DependencyInjection] removed meaningless else statement

### PR DESCRIPTION
**Pull request by issei-m**
| Q | A |
| --- | --- |
| Branch | 2.3 |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Component/DependencyInjection/Container.php [hunk 1]
```diff
@@ -238,9 +238,9 @@ public function has($id)
             }
             if (--$i && $id !== $lcId = strtolower($id)) {
                 $id = $lcId;
-            } else {
-                return method_exists($this, 'get'.strtr($id, $this->underscoreMap).'Service');
             }
+
+            return method_exists($this, 'get'.strtr($id, $this->underscoreMap).'Service');
```
Comment (nicolas-grekas) [inline src/Symfony/Component/DependencyInjection/Container.php]: this is not the same at all...

Comment (issei-m) [inline src/Symfony/Component/DependencyInjection/Container.php]: Uh oh, I was fool. Sorry. I'm gonna remove this change.

**Comment by fabpot** [comment]
Closing as one is not correct and for the other one, we are loosing the comment.

**Comment by issei-m** [comment]
Sorry, I'm sure it was not correct :(

**Pull request closed without merging**